docs/core/compatibility/breaking-changes.md

```diff
@@ -12,7 +12,7 @@ The following version selector provides a filtered list of breaking changes for
 You can also view individual issues that detail the breaking changes introduced in .NET Core 3.0 in the following GitHub repositories:
 
 - For .NET Core, the [dotnet/docs](https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change) repo.
-- For ASP.NET Core, the [aspnet/AspNetCore](https://github.com/aspnet/Announcements/issues?q=is%3Aissue+is%3Aopen+label%3A%22Breaking+change%22+label%3A3.0.0) repo.
+- For ASP.NET Core, the [aspnet/Announcements](https://github.com/aspnet/Announcements/issues?q=is%3Aissue+is%3Aopen+label%3A%22Breaking+change%22+label%3A3.0.0) repo.
```

Comment:
If you do the last change, then remove this one.

```suggestion
```

Reply:
The breaking changes will continue to get created in aspnet/Announcements. For that reason, I think we should leave this text.

Reply:
And continue to be ported here too? It's just that now the first sentence is slightly inaccurate since there are also ASP.NET Core breaking changes in the mix if you access https://github.com/dotnet/docs/issues?q=is%3Aopen+is%3Aissue+label%3Abreaking-change

Reply:
Yes, they'll be ported here from aspnet/Announcements.

Reply:
Ported as issues or just ported to this doc?
If the former, I'd recommend doing the suggested change since I can no longer see .NET Core issues only. But I'll leave it up to you to decide.